internal/webhook/sentinel/register.go

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package sentinel
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	// handlerName is the name of the webhook handler.
+	handlerName = "sentinel-webhook"
+	// WebhookPath is the path at which the handler should be registered.
+	webhookPath = "/webhooks/sentinel"
```

Comment:
Well, this is debatable. According to https://restfulapi.net/resource-naming/, it would make sense to use plural `webhooks` rather than singular `webhook` in REST API naming, since we would have multiple webhooks defined in the coming weeks/months. For the package naming on the other hand, using singular parent package names makes sense, since you import each package separately, so one package has no information about the other, unless explicitly specified/imported, so the path `internal/webhook/sentinel` is ok to be imported anywhere, but the HTTP API `<url>/webhooks/sentinel` is debatable. @unmarshall do you have any opinions on this?

Reply:
Thanks for the style guide that you've linked. We can keep it the way it is now.

Reply:
For the path having `/webhooks/<webhook-name>` is correct.